src/tools/opt-dist/src/bolt.rs

```diff
@@ -66,11 +66,18 @@ pub fn bolt_optimize(path: &Utf8Path, profile: &BoltProfile) -> anyhow::Result<(
         // Split function code into hot and code regions
         .arg("-split-functions")
         // Split as many basic blocks as possible
-        .arg("-split-all-cold")
```

Comment:
Should we really need to remove this optimization?

Reply:
This change was not intended. I'll revert this line and change the flags based on feedback.